src/components/Home/HowItWorks.tsx

```diff
@@ -98,208 +97,55 @@ function Step({ header, text, subText, index, inView, isLast }: StepProps) {
   const stepNumber = index + 1;
 
   return (
-    <StepWrapper key={header}>
-      <StepNumberWrapper>
-        <StepNumber
+    <div
+      className="mb-[96px] grid gap-x-[52px] last:mb-0 sm:grid-cols-[1fr,auto] lg:grid-cols-[auto,1fr,auto] lg:grid-rows-[auto]"
+      key={header}
+    >
+      <div className="col-span-1 col-start-1 row-start-2 hidden sm:block lg:row-start-1 xl:-ml-12">
+        <motion.p
+          className="grid h-[48px] w-[48px] place-items-center rounded-lg transition duration-300"
           initial={{
             backgroundColor: white,
-            color: grey700,
             border: `1px solid ${grey500}`,
           }}
           whileInView={{
             backgroundColor: red,
-            color: white,
             border: `1px solid ${red}`,
           }}
           viewport={{ amount: "all", margin: width > 1024 ? "-48px" : "0px" }}
           transition={{ duration: 0.3 }}
         >
-          0{stepNumber}
-        </StepNumber>
+          <motion.span
+            initial={{ color: grey700 }}
+            whileInView={{ color: white }}
+            viewport={{ amount: "all", margin: width > 1024 ? "-48px" : "0px" }}
+          >
```

Comment:
when scrolling through the step numbers, there was an ugly intermediate state where the text was still dark and the background was already red. this makes it so that the text changes quicker than the background, so that the text is always clean and readable